lib/js/atoms/Stepper.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+	<div class="stepper">
+		<div v-for="(step, index) in steps" :key="index" class="stepper__item">
+			<div
+				v-if="index !== 0"
+				class="stepper__separator"
+				:class="{ '-filled': step.isFilled }"
+			></div>
+			<ripple-wrapper class="stepper__ripple" :disable="!step.isClickable">
+				<icon
+					class="stepper__icon"
+					:class="{
+						'-filled': step.isFilled,
+						'-disabled': !step.isClickable,
+					}"
+					:icon="ICONS[step.iconKey]"
+					:size="ICON_SIZES.X_SMALL"
+					@click.native="onStepClick(step)"
+				></icon>
+			</ripple-wrapper>
+			<div class="stepper__label" :class="{ '-filled': step.isFilled }">
+				{{ step.label }}
+			</div>
+		</div>
+	</div>
+</template>
+
+<style scoped lang="scss">
+@import '../../styles/atoms/buttons';
+@import '../../styles/settings/colors';
+@import '../../styles/settings/transitions';
+@import '../../styles/settings/media-queries';
+@import '../../styles/settings/spacings';
+@import '../../styles/settings/typography';
+@import '../../styles/mixins/flex';
+
+$step-icon-size: 32px;
+
+.stepper {
+	display: flex;
+
+	&__item {
+		@include flex-center();
+	}
+
+	&__label {
+		@include textInfoM();
+		@include textBold();
+
+		color: $color-minor;
+		margin-left: $space-xxs;
+
+		&.-filled {
+			color: $color-primary;
+		}
+	}
+
+	&__icon {
+		@include flex-center();
+
+		background-color: $color-minor-background;
+		border-radius: 100%;
+		color: $color-minor;
+		height: $step-icon-size;
+		margin-right: $space-xxs;
+		width: $step-icon-size;
+		transition: background-color $basic-transition-time;
+
+		&.-filled {
+			background-color: $color-primary-background;
+			color: $color-primary;
+
+			&:hover:not(.-disabled) {
+				background-color: $color-primary-background-hovered;
```

Comment:
This color looks different in Invision. Can we double-check that?

Reply:
UX has no proposition how colors should be calculated, they agreed to stay with mix of 16% firefly-black